generate

```diff
@@ -158,7 +158,7 @@ def recurse(aspect, level=0):
                 output += "* `" + subsub_name + " <" + subsub_name + "/README.rst>`_\n"
         else:
             output += "This aspect does not have any sub aspects.\n\n"
-
+        output += "\n"
```

Comment:
What do you mean? If you're on linux, it's `./generate` and if you're on Windows, you can do `python generate`. It's a Python file...